--- ./www/webui/webui.js.orig
+++ ./www/webui/webui.js
@@ -11736,8 +11736,8 @@
       {
         var device = _this_.devices[id];
         if (device !== null && typeof(device) !== 'undefined' && device.interfaceName !== 'VirtualDevices')
-        {
-          homematic("Device.listStatus", {"id": device.id, "address": device.address, "interface": device.interfaceName}, function(data) {
+        { 
+          /* homematic("Device.listStatus", {"id": device.id, "address": device.address, "interface": device.interfaceName}, function(data) {
             if (data !== null && typeof(data) !== 'undefined')
             {
               var id = data["ID"];
@@ -11748,7 +11748,7 @@
                   device.updateStatus(data, rssiInfo);
               }
             }
-          });
+          }); */
         }
       }
     });
@@ -41506,6 +41506,7 @@
     if (this.configpendingcount === 0)
     {
       var newInputDiv = this.CreateButton("OK");
+      newInputDiv.setAttribute("id","newInputDivOK");
       newInputDiv.onclick = function()
       {
         ConfigPendingFrm.hide();
@@ -41616,6 +41617,7 @@
     tr.appendChild(td);  
 
     tbody.appendChild(tr);
+    setTimeout(function(){var elm=document.getElementById('newInputDivOK'); if (elm !== null) elm.onclick();}, 500);  
   },
 
   SetDevice: function(iface, address, direction) 
